src/mainboard/protectli/vault_ehl/gpio.h

```diff
@@ -0,0 +1,1664 @@
+
+	/* GPP_G2 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G2, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G3 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G3, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G4 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G4, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G5 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G5, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G6 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G6, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G7 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G7, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G8 - RSVD */
+	/* DW0: 0x44000b00, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_G8, NONE, DEEP, NF2),
+
+	/* GPP_G9 - RSVD */
+	/* DW0: 0x44000b00, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_G9, NONE, DEEP, NF2),
+
+	/* GPP_G10 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G10, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G11 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G11, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G12 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G12, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G13 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G13, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G14 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G14, NONE, DEEP, TxLASTRxE, SAME),
+
+	/* GPP_G15 - ESPI_IO0 */
+	/* DW0: 0x44000700, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_G15, UP_20K, DEEP, NF1),
+
+	/* GPP_G16 - ESPI_IO1 */
+	/* DW0: 0x44000702, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	PAD_CFG_NF(GPP_G16, UP_20K, DEEP, NF1),
+
+	/* GPP_G17 - ESPI_IO2 */
+	/* DW0: 0x44000700, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_G17, UP_20K, DEEP, NF1),
+
+	/* GPP_G18 - ESPI_IO3 */
+	/* DW0: 0x44000700, DW1: 0x00003000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	PAD_CFG_NF(GPP_G18, UP_20K, DEEP, NF1),
+
+	/* GPP_G19 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) - IGNORED */
+	PAD_CFG_GPIO_HI_Z(GPP_G19, NONE, DEEP, TxLASTRxE, SAME),
```

Comment:
This is SPI TPM Interrupt so it should be:
```suggestion
	PAD_CFG_GPI_APIC_LOCK(GPP_G19, NONE, LEVEL, INVERT, LOCK_CONFIG),
```